src/hotspot/os/windows/safefetch_windows.hpp

```diff
@@ -26,12 +26,13 @@
 #ifndef OS_WINDOWS_SAFEFETCH_WINDOWS_HPP
 #define OS_WINDOWS_SAFEFETCH_WINDOWS_HPP
 
+#include "sanitizers/address.hpp"
 #include "utilities/globalDefinitions.hpp"
 
 // On windows, we use structured exception handling to implement SafeFetch
 
 template <class T>
-inline T SafeFetchXX(const T* adr, T errValue) {
+ATTRIBUTE_NO_SANITIZE_ADDRESS static inline T SafeFetchXX(const T* adr, T errValue) {
```

Comment:
Wait, why is this static now?

Reply:
Habit. Removed it to be consistent with the rest of the declarations.